code/modules/pda/PDA.dm

```diff
@@ -338,31 +346,32 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	else
 		to_chat(usr, "<span class='notice'>You cannot do this while restrained.</span>")
 
-
 /obj/item/device/pda/verb/verb_remove_pen()
 	set category = "Object"
 	set name = "Remove pen"
 	set src in usr
+	remove_pen()
+
+/obj/item/device/pda/proc/remove_pen()
 
 	if(issilicon(usr))
 		return
 
 	if( can_use(usr) )
 		var/obj/item/weapon/pen/O = locate() in src
 		if(O)
+			to_chat(usr, "<span class='notice'>You remove \the [O] from \the [src].</span>")

```

Comment:
if you're going to do this, don't use `usr`. It's gross.

You should be using `user` and properly passing the argument.